src/App/AbstractApp.php

```diff
@@ -6,13 +6,15 @@
 use Dealroadshow\K8S\Framework\Core\ManifestFile;
 use Dealroadshow\K8S\Framework\Helper\Metadata\MetadataHelperInterface;
 use Dealroadshow\K8S\Framework\Helper\Names\NamesHelperInterface;
+use Dealroadshow\K8S\Framework\Project\ProjectInterface;
 
 abstract class AbstractApp implements AppInterface
 {
     protected string $appEnv;
     protected MetadataHelperInterface $metadataHelper;
     protected NamesHelperInterface $namesHelper;
     protected array $files = [];
+    protected ?ProjectInterface $project;
```

Comment:
since we support PHP8, I think it's better to use `ProjectInterface|null` everywhere, since this looks better